tests/functional/test_single_model_vehicle_attributes.py

```diff
@@ -0,0 +1,260 @@
+        - response contains proper numpy shape
+
+        """
+
+        _, ports = start_server_single_vehicle_attrib_model
+        imgs_path =  os.path.join(vehicle_attributes_data_downloader, "images")
+
+        print("imgs_path" + imgs_path)
+        img_files = os.listdir(imgs_path)
+
+        input_img = self.load_image(os.path.join(imgs_path, "021471_004.png"),72, 72)
+
+        # Connect to grpc service
+        stub = create_grpc_channel('localhost:{}'.format(ports["grpc_port"]),
+                                   PREDICTION_SERVICE)
+
+        in_name = 'input'
+        out_color = 'color'
+        out_type = 'type'
+        output = infer(input_img, input_tensor=in_name, grpc_stub=stub,
+                       model_spec_name='vehicle-attributes',
+                       model_spec_version=None,
+                       output_tensors=[out_color, out_type])
+        
+        detections_sum = 0
+        result = output[out_color]
+        print("colors:" + str(result))
+
+        result = output[out_type]
+        print("types:" + str(result))
+
+        color_prob_sum = 0.0
+        type_prob_sum = 0.0
+
+        epsilon = 0.0000001
+
+        for i in range(7):
+            color_prob_sum += output[out_color][0,i,0,0].item()
+            
+        print("color probability sum", str(color_prob_sum))
+        assert color_prob_sum + epsilon >= 1.0
+
+        print("output type shape", output[out_type].shape)
+        assert output[out_type].shape == (1, 4, 1, 1), ERROR_SHAPE
+
+        for i in range(4):
+            type_prob_sum += output[out_type][0,i,0,0].item()
+
+        print("color probability sum", str(color_prob_sum))
+        assert type_prob_sum + epsilon >= 1.0
+        assert type_prob_sum <= 1.0 + epsilon
+
+        #[car, bus, truck, van]
+        expected_type_index = 2 
+        highest_prob = 0.0
+        type_index = -1
+        for i in range(4):
+            type_prob = output[out_type][0,i,0,0].item()
+            print("type_prob " + str(type_prob))
+            if type_prob > highest_prob:
+                print("update " + str(type_prob)+ ">" +str(highest_prob))
+                highest_prob = type_prob
+                type_index = i
+
+        print("type_index= " + str(type_index))
+        assert type_index == expected_type_index
+       
+        #[white, gray, yellow, red, green, blue, black]
+        expected_color_index = 3 
+        highest_prob = 0.0
+        color_index = -1
+        for i in range(7):
+            color_prob = output[out_color][0,i,0,0].item()
+            print("color_prob " + str(color_prob))
+            if color_prob > highest_prob:
+                print("update " + str(color_prob)+ ">" +str(highest_prob))
+                highest_prob = color_prob
+                color_index = i
+
+        print("color_index= " + str(color_index))
+        assert color_index == expected_color_index
+
+
+    def test_run_inference_postprocess(self, vehicle_attributes_model_downloader,
```

Comment:
I don't think this is the right way to test it. 
Post processing itself could be tested via unit tests. 
It shall be functionally tested in a similar manner to:
https://github.com/openvinotoolkit/model_server/blob/ams/tests/functional/test_ams_inference.py
If think you can remove this test for now

Reply:
Next step will be to add functional end to end tests.